src/mgos_barometer_bme280.c

```diff
@@ -155,6 +194,22 @@ bool mgos_barometer_bme280_read(struct mgos_barometer *dev) {
   }
   dev->pressure = (float)P;
 
+// Compensatoin for humidity
+  var1 = ((double)t_fine) - 76800.0;
+  var2 = (((double)bme280_data->calib.dig_H4) * 64.0 + (((double) bme280_data->calib.dig_H5) / 16384.0) * var1);
+  var3 = Hadc - var2;
+  var4 = ((double)bme280_data->calib.dig_H2) / 65536.0;
+  var5 = (1.0 + (((double) bme280_data->calib.dig_H3) / 67108864.0) * var1);
+  var6 = 1.0 + (((double) bme280_data->calib.dig_H6) / 67108864.0) * var1 * var5;
+  var6 = var3 * var4 * (var5 * var6);
+  H = var6 * (1.0 - ((double) bme280_data->calib.dig_H1) * var6 / 524288.0);
+
+  if (H > H_max)
+      H = H_max;
+  else if (H < H_min)
+      H = H_min;
+
+  dev->humidity = (float)H;
 //  LOG(LL_DEBUG, ("P=%.2f T=%.2f", dev->pressure, dev->temperature));
```

Comment:
I know the line is commented out, but can you add humidity to the LL_DEBUG here ?